src/mail/view/MailView.js

```diff
@@ -187,8 +182,10 @@ export class MailView implements CurrentView {
 				}
 			}, [
 				m(this.viewSlider),
-				// locator.mailModel._minimizedEditors.length ? this.renderMinimizedEditors() : null
-				locator.minimizedMailModel._minimizedEditors.length? this.renderMinimizedEditors() : null
+				locator.minimizedMailModel._minimizedEditors.length ? m(MinimizedMailBar, {
```

Comment:
1. Formatting for multiline condition is wrong
2. You are using private field of minimizedMailModel. We should either make it non-private or add a getter.